driver/razerkbd_driver.h

```diff
@@ -79,6 +79,7 @@
 #define USB_DEVICE_ID_RAZER_BOOK_2020 0x026A
 #define USB_DEVICE_ID_RAZER_BLADE_15_ADV_EARLY_2021 0x026D
 #define USB_DEVICE_ID_RAZER_BLADE_15_ADV_MID_2021 0x0276
+#define USB_DEVICE_ID_RAZER_BLADE_14_2021 0x0270
```

Comment:
Move it one up to keep it sorted

Reply:
@z3ntu  @BeanieBen9990  Made a pull request for this change.